openmetadata-ui/src/main/resources/ui/src/components/AddService/Steps/SelectServiceType.tsx

```diff
@@ -32,12 +33,25 @@ const SelectServiceType = ({
   onNext,
 }: SelectServiceTypeProps) => {
   const [category, setCategory] = useState('');
+  const [connectorSearchTerm, setConnectorSearchTerm] = useState('');
+  const [selectedConnectors, setSelectedConnectors] = useState<string[]>([]);
+
+  const handleConnectorSearchTerm = (value: string) => {
+    setConnectorSearchTerm(value);
+    setSelectedConnectors(
+      serviceTypes[serviceCategory].filter((c) =>
+        c.toLowerCase().includes(value.toLowerCase())
```

Comment:
Can we use `toLower` from lodash here?

Reply:
I think it won't make any difference for this case, and I think the in-build method will run faster compare to lodash. let me know if I am wrong here!